net-libs/cppzmq/cppzmq-4.6.0.ebuild

```diff
@@ -0,0 +1,42 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake
+
+DESCRIPTION="High-level CPP Binding for ZeroMQ"
+HOMEPAGE="https://github.com/zeromq/cppzmq"
+SRC_URI="https://github.com/zeromq/${PN}/archive/v${PV}.tar.gz ->  ${P}.tar.gz"
+
+SLOT="0"
+LICENSE="MIT"
+KEYWORDS="~amd64 ~x86 ~x86-linux"
+
+# Except for test these are used to alter installed cmake files:
+IUSE="test drafts static-libs"
+RESTRICT="!test? ( test )"
+
+DEPEND=">=net-libs/zeromq-4.3.1[drafts?,static-libs?]
+	test? ( dev-cpp/catch )"
+RDEPEND=">=net-libs/zeromq-4.3.1[drafts?,static-libs?]"
```

Comment:
This looks like 1:1 repeat. Why not swap them and use RDEPEND in DEPEND?

Reply:
ok, now they are different